R/tbl_cross.R

```diff
@@ -0,0 +1,163 @@
+#' Create a cross table of summary statistics
+#'
+#' Wrapper for [tbl_summary] to cross tabulate two variables
+#'
+#' @param data A data frame
```

Comment:
Add `@inheritParams tbl_summary` to inherit parameter descriptions that are the same in the two functions